frontend/public/components/utils/link.tsx

```diff
@@ -66,7 +66,7 @@ export const getURLSearchParams = () => {
   return all;
 };
 
-export const ExternalLink: React.FC<ExternalLinkProps> = ({href, text, additionalClassName=''}) => <a className={classNames('co-external-link', additionalClassName)} href={href} target="_blank" rel="noopener noreferrer">{text}</a>;
+export const ExternalLink: React.FC<ExternalLinkProps> = ({href, text, additionalClassName='', dataTestID=''}) => <a className={classNames('co-external-link', additionalClassName)} href={href} target="_blank" rel="noopener noreferrer" data-test-id={dataTestID}>{text}</a>;
```

Comment:
Does this mean there will be a `data-test-id=""` attribute on every link? Should we leave this `undefined` if not set? (I'm not sure React's behavior offhand here.